.gitignore

```diff
@@ -0,0 +1,2 @@
+test.md
+app_screenshot.png
```

Comment:
- [ ] [_Optional_] Add `node_modules/` aswell in the .gitignore so they never get pushed to the repo, even by mistake.